core/vidl/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ IF( FFMPEG_FOUND )
                                      vidl_ffmpeg_ostream_v4.txx
       vidl_ffmpeg_istream_v0.9.txx   vidl_ffmpeg_ostream_v0.9.txx
                                      vidl_ffmpeg_ostream_v0.11.txx
+      vidl_ffmpeg_istream_v56.txx    vidl_ffmpeg_ostream_v56.txx
```

Comment:
The naming convention for these files is based on the ffmpeg major/minor version rather than the libavcodec version number.  So these files should be named vidl_ffmpeg_*stream_v2.8.txt.  You can see that there are lots of gaps in supported versions here.  We haven't updated this since FFMPEG v0.11!

The exception here (which is confusing) are the files numbered v1, v2, v3, v4.  This was an arbitrary numbering we used before FFMPEG was making regular versioned releases.  Long ago the FFMPEG version rarely changed (I think it was v0.5), but they keep making changes to the API and Linux distros would pick up those and release them.  So the original file naming was arbitrary.  At this point we should probably either throw away those original version or renamed them in a better way (maybe v1beta.txt or something).

The reason not to use the libavcodec version number here is that we wanted to make it more clear which versions of FFMPEG were supported by looking at the files.  Also we are checking both libavformat and libavcodec versions in different parts of the code and the relationship between each of these and FFMPEG version is not clear.  That said, maybe it would be cleaner to just use libavcodec and libavformat versions to name the ostream and istream files respectively.